index.js

```diff
@@ -1,25 +1,42 @@
 'use strict';
 
+const modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
+  strict: false,
+}];
+
 module.exports = function buildAirbnbPreset(context, options) {
-  var preset; // eslint-disable-line no-var
-  if (options && options.modules === false) {
-    preset = require('babel-preset-es2015').buildPreset(null, { modules: false });
-  } else {
-    preset = require('babel-preset-es2015-without-strict');
-  }
   return {
     presets: [
-      preset,
+      require('babel-preset-env').default(null, {
+        debug: true,
```

Comment:
https://github.com/babel/babel-preset-env#debug

console.logs the plugins being used and the browser versions that are being applied to each plugin.